bundle/View/Redirect/Resolver.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Netgen\Bundle\EzPlatformSiteApiBundle\View\Redirect;
+
+use Netgen\Bundle\EzPlatformSiteApiBundle\View\ContentView;
+use Netgen\EzPlatformSiteApi\API\Values\Content;
+use Netgen\EzPlatformSiteApi\API\Values\Location;
+use Symfony\Component\Routing\RouterInterface;
+
+final class Resolver
+{
+    /**
+     * @var \Netgen\Bundle\EzPlatformSiteApiBundle\QueryType\ParameterProcessor
+     */
+    private $parameterProcessor;
+
+    /**
+     * @var \Symfony\Component\Routing\RouterInterface
+     */
+    private $router;
+
+    /**
+     * Resolver constructor.
+     *
+     * @param \Netgen\Bundle\EzPlatformSiteApiBundle\View\Redirect\ParameterProcessor $parameterProcessor
+     * @param \Symfony\Component\Routing\RouterInterface $router
+     */
+    public function __construct(
+        ParameterProcessor $parameterProcessor,
+        RouterInterface $router
+    ) {
+        $this->parameterProcessor = $parameterProcessor;
+        $this->router = $router;
+    }
+
+    /**
+     * Builds a path to the redirect target.
+     *
+     * Value from configuration can be:
+     *      - SiteAPI location
+     *      - SiteAPI content
+     *      - location id fetched from config resolver
```

Comment:
content ID could be added also, which can generate a link to a main location.

Reply:
Not sure what you mean here? How would that configuration look like?
I'm talking here about this:
`temporary_redirect: "@=configResolver.getParameter("some_location_id", "ngsite")"`

Maybe rewording of the docs is needed here?

Reply:
What I meant is, I'd definitelly expect to be able to also use a content ID for redirection too. There is no place where it's documented that `temporary_redirect: "@=configResolver.getParameter("some_location_id", "ngsite")"` works, but `temporary_redirect: "@=configResolver.getParameter("some_content_id", "ngsite")"` does not work.

The only place where that is apparent is in the resolver using the hardcoded `locationId` parameter to `ez_url_alias` route.

Reply:
Besides, you don't have to use config resolver to provide the ID. You could just as well use `temporary_redirect: 42` and it would work. Is it a location ID or a content ID?

Reply:
It was internally agreed to just remove support for IDs as targets, too much ambiguity to begin with.